master:    
  file_path: "/tmp/" # in home folder
  
  search_method: "random" # random, grid, fixed, ...
  number_of_sets_to_test: 10000000 # only for random search, 0 for unlimited

  parameters:
    freq:   
        min: 0.2
        max: 3
        step_size: 0.1
    double_support_ratio:
        min: 0.0
        max: 0.5
        step_size: 0.1
    foot_rise:
        min: 0.07
        max: 0.1
        step_size: 0.01
    trunk_swing:
        min: 0.0
        max: 1.0
        step_size: 0.1
    trunk_height:
        min: 0.33
        max: 0.48
        step_size: 0.05
    trunk_pitch:
        min: -0.2
        max: 0.4
        step_size: 0.1
    trunk_phase:
        min: -0.5
        max: 0.5
        step_size: 0.1
    foot_overshoot_ratio:
        min: 0.0
        max: 0.5
        step: 0.1
    foot_overshoot_phase:
        min: 0.7
        max: 1.0
        step: 0.1
    foot_apex_phase:
        min: 0.3
        max: 0.7
        step: 0.1
    trunk_x_offset:
        min: -0.05
        max: 0.05
        step: 0.0
